docs/documentation/tabs-explained/tab-home.md

```
@@ -1,6 +1,16 @@
+---
+description: This page explains how a user can load an EPW file in the Clima tool
+---
+
 # Select Weather File
 
-Users can either choose to analyse the climate of the locations displayed on the map or upload a custom EPW file. After loading an EPW file the user can then access the other tabs to generate dynamic visualisations of the data.
+Users can either choose to analyse the climate of the locations displayed on the map or upload a custom EPW file. After loading an EPW file the user can then access the other tabs to generate dynamic visualisations of the data.&#x20;
+
+### Video Tutorial
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)